source/ddbus/thin.d

```diff
@@ -696,6 +701,21 @@ struct DBusAny {
       return uint64 == b.uint64;
     }
   }
+
+  /// Returns: true if this variant is an integer.
+  bool typeIsIntegral() const @property {
+    return type == 'y' || type == 'n' || type == 'q' || type == 'i' || type == 'u' || type == 'x' || type == 't';
+  }
+
+  /// Returns: true if this variant is a floating point value.
+  bool typeIsFloating() const @property {
+    return type == 'd';
+  }
+
+  /// Returns: true if this variant is an integer or a floating point value.
+  bool typeIsNumeric() const @property {
+    return typeIsIntegral || typeIsFloating;
+  }
```

Comment:
I think these are more at home in utils.d

Reply:
Hm I would agree with that but utils isn't imported with `import ddbus;` so using it would require a second import and it feels like you are importing internal utilities there, even though these types are meant for regular usage and probably will only be used with these variants.